canvas_sdk/protocols/clinical_quality_measure.py

```diff
@@ -23,6 +25,10 @@ class Meta:
         is_abstract: bool = False
         is_predictive: bool = False
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        self.now = arrow.utcnow()  # TODO now should be local time
```

Comment:
@aduane @jrwils There is a `now` member on ClinicalQualityMeasure in the workflow kit so that "now" can be established for the entire duration of a plugin run, rather than using different datetimes from each time `arrow.get` is called.

I brought this over for use in plugins.

There is more stuff (like establishing a default timeframe) in the `__init__` that we probably want to bring over as well eventually.